src/modules/Packages.rb

```diff
@@ -134,6 +139,8 @@ def main
       @base_source_id = nil
 
       @old_packages_proposal = nil
+
+      PackageSystem.EnsureSourceInit
```

Comment:
This initialization is not needed here, it's done outside of the module by the installation framework. You need this only when playing with the package management in a separate script...